exercises/concept/elyses-analytic-enchantments/.docs/hints.md

```diff
@@ -0,0 +1,40 @@
+# Hints
+
+## 1. Determine if a card is present
+
+- `Vector` indices start at `1`.
```

Comment:
Not sure why this hint is here? Maybe it could be moved above subsection 1 as a generic hint?

Reply:
The original exercise has "find the position of a card" as first task, I mixed things up when forking it.